privacyidea/api/periodictask.py

```diff
@@ -106,6 +106,7 @@ def list_periodic_tasks():
 
 @periodictask_blueprint.route('/<ptaskid>', methods=['GET'])
 @log_with(log)
+@prepolicy(check_base_action, request, ACTION.PERIODICTASKREAD)
```

Comment:
We should also add this decorator to ``list_periodic_tasks``.

Reply:
Who named the function ``list_periodic_tasks`` in the first place!?
But you are right. Will do.

Reply:
done in 003f1a44
(might be a good idea to resolve conversations)